redun/job_array.py

```diff
@@ -78,7 +80,7 @@ class JobArrayer:
 
     def __init__(
         self,
-        executor: "AWSBatchExecutor",
+        executor: Any,
```

Comment:
This is a sign the interface here could be better. For now we can use this work around.